spec/models/user_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before(:all) do
+    @user = User.create(name: 'test', photo: 'https://unsplash.com/photos/', bio: 'test')
+  end
+
+  it 'should create a user' do
+    expect(@user).to be_valid
+  end
+end
```

Comment:
- [ ] Kindly add test for the `latest_posts` method in the Post model.

- [ ] Kindly add tests for the name and posts counter validations.